src/arch/MovieTexture/MovieTexture_Generic.h

```diff
@@ -108,7 +108,7 @@ class MovieTexture_Generic: public RageMovieTexture
 	static EffectMode GetEffectMode( MovieDecoderPixelFormatYCbCr fmt );
 
 private:
-	MovieDecoder *decoder_;
```

Comment:
In a future PR can we opt to use unique_ptrs instead of raw ptrs (assuming this class has ownership)?

Reply:
I actually started doing it, but felt it was a bit tangent to fixing the looping logic. I can for sure can change it in a follow up.

Reply:
Yeah follow up seems fine.